src/libsync/owncloudtheme.cpp

```diff
@@ -45,8 +45,9 @@ QString ownCloudTheme::about() const
 {
     QString devString;
     devString = trUtf8("<p>Version %2. For more information visit <a href=\"%3\">%4</a></p>"
-               "<p><small>By Klaas Freitag, Daniel Molkentin, Jan-Christoph Borchardt, "
-               "Olivier Goffart, Markus Götz and others.</small></p>"
+               "<p>For known issues and help, please visit: <a href=https://central.owncloud.org/c/help/desktop-file-sync>central.owncloud.org</a></p>"
+               "<p><small>By Klaas Freitag, Daniel Molkentin, Olivier Goffart, Markus Götz, "
```

Comment:
@guruz i get the line but i just can't figure out the right way... could you please point me somewhere for an example?
Thanks!